concrete/src/Application/Service/Dashboard.php

```diff
@@ -247,6 +247,8 @@ public function getDashboardBackgroundImage()
      */
     public function getIntelligentSearchMenu()
     {
+        $loc = Localization::getInstance();
+        $loc->setActiveContext('system');
```

Comment:
Another thing is that we should call `$loc->pushActiveContext(Localization::CONTEXT_UI)` and then later on when the rendering is finished call `$loc->popActiveContext()`.

This is because this element is rendered as part of the actual page (in the page footer), there might be some strings that are rendered after this element that are expecting the active context to be correctly set.